htdocs/core/class/html.formadmin.class.php

```diff
@@ -125,6 +125,7 @@ public function select_language($selected = '', $htmlname = 'lang_id', $showauto
 		{
 			include_once DOL_DOCUMENT_ROOT.'/core/lib/ajax.lib.php';
 			$out .= ajax_combobox($htmlname);
```

Comment:
ajax_combobox() is a generic component that should works with all combolist.
Can you describe how to reproduce the bug ?

Reply:
Create a thirdparty and set its default language to any value that is not your default language. Then click on modify in the same thirdparty and save without changing anything. The default language now should be something else than you selected. In the standard html select, the correct value is set but in the select2 input the value is different.

Reply:
I had this bug with system wide default language 'de_DE' and the thirdparty default language 'en_GB'

Reply:
Problem should come from somewhere else.
I can't reproduce with 12.0.3. Did you ?

Reply:
I have this error in 12.0.2. I currently have no capacities to test with 12.0.3 but will try it as soon as possible